sequencing/vcf_qc/dxapp.json

```diff
@@ -3,35 +3,108 @@
   "title": "VCF QC",
   "summary": "Apply Recalibration and hard filters to VCF file",
   "dxapi": "1.0.0",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
-      "name": "vcf_fn",
+      "name": "fn_vcfgz",
```

Comment:
Is this our standard name for VCF files?

Reply:
Not quite. We will discuss this issue on July 11.

Reply:
"fn_vcfgz" -> "variants_vcfgzs"